### PR TITLE
files changed vite logo deleted

### PR DESCRIPTION
**Pull request by mtabba**
App.css changes made. 
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,4 +11,3 @@
     <script type="module" src="/src/main.jsx"></script>
   </body>
 </html>
-<!-- Testing -->
```
Comment (waqar-ahmedev) [inline index.html:14]: Leave an empty line at the end.

File: src/App.css [hunk 2]
```diff
@@ -45,10 +45,6 @@ button:focus-visible {
   }
 }
 
-
-
-/* App.css */
-
```
Comment (waqar-ahmedev) [inline src/App.css:51]: Leave an empty line at the end.

Comment (waqar-ahmedev) [reply src/App.css:51]: Leave an empty line at the end.

Comment (waqar-ahmedev) [reply src/App.css:51]: Ok Theek hai.

File: index.html [hunk 3]
```diff
@@ -11,4 +11,3 @@
     <script type="module" src="/src/main.jsx"></script>
   </body>
 </html>
```
Comment (waqar-ahmedev) [inline index.html:13]: Yeh to ok Hai.

File: index.html (hunk 1 above)
Comment (mtabba) [reply index.html:14]: this was old file. file has been changed now

**Pull request merged**